common/src/main/java/org/duracloud/mill/config/ConfigConstants.java

```diff
@@ -31,6 +31,8 @@
      */
     public static final String WORK_DIRECTORY_PATH = "workdir";
     public static final String NOTIFICATION_RECIPIENTS = "notification.recipients";
+    public static final String NOTIFICATION_RECIPIENTS_NON_TECH = "notification.recipients.non-tech";
+    public static final String NOTIFICATION_SENDER = "notification.sender";
```

Comment:
These two new properties should be added to the /resources/mill-config-sample.properties file